src/utils/tokens.ts

```diff
@@ -130,14 +143,20 @@ const tokens = {
    * Calculate deposit value for the given token mint amount
    *
    * @param {number} mintAmount Amount of tokens being minted
+   * @param {boolean} isCreateNFT If we are calculating the deposit for a NFT creation
    *
    * @return {number}
    * @memberof Tokens
    * @inner
    *
    */
-  getDepositAmount(mintAmount: number): number {
-    return Math.ceil(TOKEN_DEPOSIT_PERCENTAGE * mintAmount);
+  getDepositAmount(mintAmount: number, isCreateNFT: boolean = false): number {
```

Comment:
Because we had a `getMintDepositInfo` which had a `isNFT` parameter and it called `getDepositAmount` then added 1 if `isNFT` was true.
I put this logic inside the `getDepositAmount` so we would not need to replicate it in other places where we need to calculate deposit amount with nfts.

Reply:
We can keep it to make the code easier to read but the docstring must explain this in details. The deposit amount usually you can get back but this +1 from the NFT is a fee, not a deposit, that's my main concern.